src/db-connection.js

```diff
@@ -57,6 +67,18 @@ function _prepareLearningContentValueBeforeInsertion(learningContentItem, fieldS
   return fieldStructure.isArray ? `{${value.join(',')}}` : value[0];
 }
 
+function _allValuesHavePrimaryKey(values) {
+  return !values.some(function(value) {
+    const primaryKey = value[0];
+    return primaryKey == null;
```

Comment:
Ce ne serait pas mieux d'utiliser l'écriture `return !!primaryKey` pour tenir compte du cas `undefined` ?

Reply:
@jbuget tu veux dire:
```suggestion
    return !primaryKey;
``` 
?
J'aimerais que ça retourne true si c'est `null` ou `undefined`.